src/routes/assistants/+page.svelte

```diff
@@ -129,16 +148,32 @@
 					<a
 						href={getHref($page.url, {
 							newKeys: { user: data.user.username },
-							existingKeys: { behaviour: "delete", keys: ["modelId", "p"] },
+							existingKeys: { behaviour: "delete", keys: ["modelId", "p", "q"] },
 						})}
-						class="flex items-center gap-1.5 rounded-full border px-3 py-1 {assistantsCreator &&
+						class="flex items-center gap-1.5 truncate rounded-full border px-3 py-1 {assistantsCreator &&
 						createdByMe
 							? 'border-gray-300 bg-gray-50  dark:border-gray-600 dark:bg-gray-700 dark:text-white'
 							: 'border-transparent text-gray-400 hover:text-gray-800 dark:hover:text-gray-300'}"
 						>{data.user.username}
 					</a>
 				{/if}
 			{/if}
+			<div
+				class="ml-auto mr-2 flex h-9 w-9 items-center gap-x-2 rounded-full border border-gray-300 px-2 transition-all duration-300 ease-out hover:w-64 md:w-64 dark:border-gray-500"
+			>
+				<button class="flex-shrink-0 cursor-default" on:click={() => filterInputEl.focus()}
+					><CarbonSearch class="text-gray-400" /></button
+				>
+				<input
```

Comment:
I get weird behaviours:

https://github.com/huggingface/chat-ui/assets/3841370/a1f51c27-96b3-4c08-b18d-d25fe71b2792

Reply:
when you load the page, you see `undefined` as the value in the box ?

Reply:
https://github.com/huggingface/chat-ui/pull/841/commits/c0387ce8eb8bcb8da9a002a5d5d3531a6e67154f trimming the value was turning `null` into `undefined`

shoud be fixed by here https://github.com/huggingface/chat-ui/pull/841/commits/d65f7c17ecb4e904e6dfff0e6e83d08ab079ff16